### ISSUE TITLE
Entry file is huge in npm

### ISSUE THREAD

**Issue by esamattis** [Bartozzz/get-link#34]
```
$ ls -lh node_modules/get-link/dist/index.js
-rw-r--r--  1 esamatti  staff    89K Oct 26  1985 node_modules/get-link/dist/index.js
```

It seem to contain babel polyfill which should not be distributed in npm.

But it seems that 4ecd64642b67568e88092ca88434e9dc12d0fe10 fixes this. Can you make a new release?

**Comment by Bartozzz** [comment]
Sure, sorry for that. I'll make a new release as soon as possible.

**Comment by Bartozzz** [comment]
I've published `v1.0.4` on [npm](https://www.npmjs.com/package/get-link). You can install it with the following command:

```bash
$ npm i --save get-link@1.0.4
```

It reduces package size significantly & includes your pull request (#33).

**Comment by esamattis** [comment]
Thank you!
